keyboards/lets_split_eh/keymaps/romus/rules.mk

```diff
@@ -0,0 +1,11 @@
+# Build options
+
+BACKLIGHT_ENABLE = no		# Switch LEDs
+MOUSEKEY_ENABLE = yes		# Emulates mouse key using keypresses
+RGBLIGHT_ENABLE = yes 		# LED strips
+TAP_DANCE_ENABLE = no 		# Use multi-tap features
+AUDIO_ENABLE = no 			# Audio stuff
+
+ifndef QUANTUM_DIR
```

Comment:
Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.